client/src/fuzzy-search.ts

```diff
@@ -60,7 +60,11 @@ export class FuzzySearch {
     // each match's ranges so we can finally result a list of sub-strings.
     const res: CombinedMatchResult[] = candidates
       .map((doc) => {
-        const matched = match(needle, doc.url, {
+        // The needle will always start with a `/` and so will always the `doc.url`
+        // but it's not interesting to match on it. So, essentially,
+        // match `wboo` with `en-US/docs/Web/Foo`
+        // instead of `/wboo` with `/en-US/docs/Web/Foo`
+        const matched = match(needle.slice(1), doc.url.slice(1), {
```

Comment:
Ah so I was confused because you didn't revert my changes to `search.test.js` so I didn't realize the old test now passes with your changes. Looks like the key difference is here where you remove the leading slash from the url while I only remove it from the query.